app/assets/javascripts/main.js.erb

```diff
@@ -468,16 +525,17 @@ $(function() {
             type: 'GET',
             url: '/info_window',
             data: {
-              'thing_id': activeThingId,
+              'thing_id': activeThingId
+              /*
               'flash': {
```

Comment:
Same as below, could you add `*`s to the commented lines to visually make them stand out?